WalletWasabi.Gui/Config.cs

```diff
@@ -44,29 +44,21 @@ public class Config : IConfig
 		[JsonProperty(PropertyName = "UseTor")]
 		public bool? UseTor { get; internal set; }
 
-		[JsonProperty(PropertyName = "TorHost")]
-		public string TorHost { get; internal set; }
+		[JsonProperty(PropertyName = "TorSocks5EndPoint")]
```

Comment:
I suggest to use `nameof` in all these attributes/decoration just to make sure we are not making mistakes. So, instead of:
```suggestion
		[JsonProperty(PropertyName = nameof(TorSocks5EndPoint))]
```

Reply:
If accidentally somebody change the Property name the Json will still work. If we use nameof that won't be true. So PropertyName is an additional barrier to not mess up the json token names.